### PR TITLE
fix(clayui.com): Some documents have wrong ToC jump

### PR DESCRIPTION
**Pull request by pat270**
#2814

The only one not fixed on that list is #2797

### CODE REVIEW

File: clayui.com/content/docs/components/css-management-toolbar.md [hunk 1]
```diff
@@ -1326,12 +1326,7 @@ Use `navbar-overlay navbar-overlay-up` on any direct descendant of navbar to cre
 </nav>
 ```
 
--   [Search](#search)
-    -   [Summary](#summary)
-    -   [Results](#results)
-    -   [Results with Filter](#results-with-filter)
-
-## Search
+## Search(#css-markup-search)
```
Comment (bryceosterhaus) [inline clayui.com/content/docs/components/css-management-toolbar.md]: This syntax is a great add, nice job.

**Comment by pat270** [comment]
Should be fixed now, thanks for help @matuzalemsteles

**Comment by matuzalemsteles** [comment]
Just started reviewing :)

**Comment by matuzalemsteles** [comment]
Resent #2878

@pat270 I am resending this, apparently there was another SF error that was breaking the CI but it was not visible.

**Pull request closed without merging**